richardcoffey/minecraft-website
# Style and DOM Improvements

This code improves the structure of the `style.css` file, and improves headers styling. It also adds the background image into the repo, and styles the body through css instead of with dom object properties.

Steps to test:
* Render each HTML page.
* Ensure that the header text is white, and enlarged.
* Ensure that the background image renders.
